### PR TITLE
Fetch submodules when doing cross-arch tests

### PR DESCRIPTION
**Pull request by llucax**
We need to do that because if the project has submodules, they need to be available when running the tests in the container too.

### CODE REVIEW

**Comment by llucax** [comment]
Enabled auto-merge.

**Review by matthias-wende-frequenz — DISMISSED** [review]

**Comment by llucax** [comment]
Rebased to resolve some conflicts, needs a new approval @matthias-wende-frequenz

**Review by leandro-lucarella-frequenz — DISMISSED** [review]

**Comment by llucax** [comment]
Self-approving :grimacing:

**Comment by llucax** [comment]
It doesn't work, either I have to force merge or wait for another approval.

**Pull request merged**
